### PR TITLE
fix up for #5

### PR DESCRIPTION
**Pull request by SamYuan1990**
em... if I remember correctly,  some missing files are auto generated by `build.gradle`
So far:
1) commit missing files 1st to make the ci works.
2) fix for logic and check style errors. 

Signed-off-by: SamYuan1990 <yy19902439@126.com>

### CODE REVIEW

File: src/main/java/SM3Util.java [hunk 1]
```diff
@@ -38,10 +42,6 @@
      */
     public static boolean verify(byte[] srcData, byte[] sm3HashVal) {
         byte[] hashVal = hash(srcData);
-        if (Arrays.equals(hashVal, sm3HashVal)) {
```
Comment (SamYuan1990) [inline src/main/java/SM3Util.java:41]: @davidkhala , pls check here

Comment (davidkhala) [reply src/main/java/SM3Util.java:41]: How can I help? Do you mean checking whether your PR code is correct or the function used here is propriate?

Comment (SamYuan1990) [reply src/main/java/SM3Util.java:41]: I suppose here is the only logic replacement change, others are import orderer and static var rename, etc changes.
By checkstyle, it says the changes here(before/after) are the same.
If needed, please contract original author for this code line for double confirm?

File: src/main/java/SM4Util.java [hunk 2]
```diff
@@ -16,7 +16,11 @@
  */
 public class SM4Util {
 
-    private final static String ALGORITHM_NAME = "SM4";
+    private static String algorithmName = "SM4";
```
Comment (davidkhala) [inline src/main/java/SM4Util.java:19]: I think it could be moved to a constants definition file or somewhere else, as public.

Comment (SamYuan1990) [reply src/main/java/SM4Util.java:19]: agree, so far in this pr, just making it pass checkstyle.

**Pull request merged**